File: MyPlanner.Clients/React/src/entities/TodoTask.ts
export default class TodoTask{
    id : string;
    title: string;
    description: string = "";
    isComplete: boolean = false;
    listId : string;
    constructor(id: string, title: string, listId : string, description: string = "", isComplete: boolean = false){
        this.id = id;
        this.title = title;
        this.listId = listId;
        this.description = description;
        this.isComplete = isComplete;
    }
}

File: MyPlanner.Data/Entities/Todo/TodoTask.cs
﻿namespace MyPlanner.Data.Entities.Todo;

public class TodoTask : EntityBase
{
    public string Title { get; set; } = string.Empty;
    public string Description { get; set; } = string.Empty;
    public bool IsComplete { get; set; }
    public Guid ListId { get; set; }
    public List<TodoTaskSession> Sessions { get; set; } = new List<TodoTaskSession>();
}

